config.json

```diff
@@ -1430,6 +1430,18 @@
         "strings"
       ],
       "deprecated": true
+    },
+    {
+      "slug": "grep",
+      "uuid": "78bcbae1-a0f2-460c-ba89-e51844fe9397",
+      "core": false,
+      "unlocked_by": null,
+      "difficulty": 4,
+      "topics": [
```

Comment:
Can you confirm that all three are valid topics from the TOPICS.txt list?

Reply:
Well i got the topics from the python track so i guess there must be valid => https://github.com/exercism/python/blob/master/config.json